buildscripts/run_examples.py

```diff
@@ -56,7 +65,8 @@ def run_examples(sdc_utils):
             passed += 1
             sdc_utils.log_info(f'{sdc_example} PASSED')
 
-    sdc_utils.log_info(f'SDC examples summary: {total} RUN, {passed} PASSED, {failed} FAILED', separate=True)
+    summary_msg = f'SDC examples summary: {total} RUN, {passed} PASSED, {failed} FAILED, {skipped} SKIPPED'
+    sdc_utils.log_info(summary_msg, separate=True)
```

Comment:
@densmirn ,
I would prefer not to skip any example - they all should be executed and all should be passed.
To add sklearn (is it scikit-learn??) to CI for example build just add required package to https://github.com/IntelPython/sdc/pull/761/files#diff-12db1f3244c35f97542b6b38613320cfL82
sdc_utils.create_environment(['scipy'])

Reply:
Like sdc_utils.create_environment(['scipy', 'sklearn'])

Reply:
Done!

Reply:
I hope it won't slow down CI.

Reply:
I skipped the example again, because the example uses big external dataset as input. So we are not able to run it in CI. This example is just for documentation and successfully works with downloaded dataset on the local system.

Reply:
@Vyacheslav-Smirnov are you ok with that? If yes, please re-review the PR, if not, let's think up about another solution, but I don't think we have enough time before release.

Reply:
I just do not like the idea of skipping examples in CI) They all should work.
My thought is - can we just include the example in doc without adding it in examples?
Or maybe there is some less restrictive example?
If none of above suits - then lets merge this PR for now because the documentation need to be completed.

Reply:
I would like to let user run the example directly from repo, so the file should be located there. Maybe we can think up about less restrictive example in the future. Now I prefer to merge the PR. Thanks!